apps/serdes/.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore halide serialized buffer files
+*.hlb
```

Comment:
Not at all important right now, but it's been a long time since ~any OS we care about was limited to three-character file extensions; something a bit less terse might be nice. (e.g., TFLite uses `.tflite` as an extension, which is a lot easier to read IMHO)

Reply:
I propose using `.hlpipe` (Halide pipeline)